internal/docs/jsp/JSPTCD.txt

```diff
@@ -65,7 +65,7 @@ Quality Assurance
 
 Justification of Adequacy
 -------------------------
-The JavaServer Pages and Expression Language tests, distributed as part 
+The Jakarta Server Pages and Expression Language tests, distributed as part 
 of the JavaEE CTS version 7, provides an adequate set of tests to ensure all
```

Comment:
`JavaEE CTS version 7` needs update

Reply:
Fixed. Along with the Java EE reference on the next line.